netbox/ipam/tests/test_ordering.py

```diff
@@ -0,0 +1,290 @@
+from django.test import TestCase
+
+from ipam.choices import IPAddressStatusChoices, PrefixStatusChoices
+from ipam.models import IPAddress, Prefix, VRF
+
+import netaddr
+
+
+class PrefixOrderingTestCase(TestCase):
+    vrfs = None
+
+    def setUp(self):
+        vrfa = VRF(name="VRF A")
+        vrfb = VRF(name="VRF B")
+        vrfc = VRF(name="VRF C")
+        VRF.objects.bulk_create([vrfa, vrfb, vrfc])
+        self.vrfs = (vrfa, vrfb, vrfc)
+
+    # Function to create all prefixes
+    def _create_prefix(self, prefixes):
```

Comment:
Two issues with this:

1. `prefix` implies a singular instance, but it deals with a list of instances.
2. It's not actually _creating_ the prefixes, just instantiating them.

Something like `prefix_from_args()` would be more accurate. IMO it would be fine to have this deal with a single object, and call it inside list comprehension during creation:

```
Prefix.objects.bulk_create([prefix_from_args(args) for args in prefixes])
```